contrib.html

```diff
@@ -97,7 +97,7 @@ <h2>Contribute</h2>
             <div class="row">
                 <div class="col-lg-12">
                     <hr>
-                    <p>Copyright © 2014-2016 Red Hat Inc. and Avocado Community Contributors</p>
+                    <p>Copyright © 2014-2020 Red Hat Inc. and Avocado Community Contributors</p>
```

Comment:
Just a nit-pick for the future: The copyrigh modification is not a modification by itself. Copyright should only be updated when there is an active development and to be precise one should only list years where there are significant changes, ideally together with the change itself. I can only see a minor modifications/bugfixes which should not be covered by a copyright. Anyway I agree that in many cases this is being done regularly so take it just for your consideration.